src/main/java/com/solvd/pages/common/ProductListPageBase.java

```diff
@@ -0,0 +1,37 @@
+package com.solvd.pages.common;
+
+import java.util.List;
+import com.solvd.pages.SortType;
+import org.openqa.selenium.WebDriver;
+
+public abstract class ProductListPageBase extends PageBase {
+    public ProductListPageBase(WebDriver driver) {
```

Comment:
it's a good practice to leave a blank line between the class declaration and the first field